test/conftest.py

```diff
@@ -6,6 +6,8 @@
 
 here = os.path.abspath(os.path.dirname(__file__))
 
+pytest_plugins = ["test.fixtures"]
```

Comment:
I think you might as well just inline this function into `conftest.py`. People will be more likely to look here for fixtures, as it's the standard spot to put them.